diff --git a/src/gplot.c b/src/gplot.c
index 276d583..26512fb 100644
--- a/src/gplot.c
+++ b/src/gplot.c
@@ -396,7 +396,7 @@ l_int32  ignore;
        snprintf(buf, L_BUF_SIZE,
                "wgnuplot -persist %s", gplot->cmdname);
 #endif  /* _WIN32 */
-    ignore = system(buf);  /* gnuplot || wgnuplot */
+    //ignore = system(buf);  /* gnuplot || wgnuplot */
     return 0;
 }
 
diff --git a/src/viewfiles.c b/src/viewfiles.c
index 4860eb4..8fab2a7 100644
--- a/src/viewfiles.c
+++ b/src/viewfiles.c
@@ -32,6 +32,7 @@
  */
 
 #include <string.h>
+#include <sys/errno.h>
 #include "allheaders.h"
 
 #ifdef _WIN32
@@ -117,7 +118,8 @@ SARRAY    *safiles, *sathumbs, *saviews, *sahtml, *salink;
         /* Make the output directory if it doesn't already exist */
 #ifndef _WIN32
     sprintf(charbuf, "mkdir -p %s", dirout);
-    ret = system(charbuf);
+    //ret = system(charbuf);
+	ret = ENOENT;	// For iOS compat, claim ENOENT
 #else
     ret = CreateDirectory(dirout, NULL) ? 0 : 1;
 #endif  /* !_WIN32 */
diff --git a/src/writefile.c b/src/writefile.c
index e72645a..41d86d9 100644
--- a/src/writefile.c
+++ b/src/writefile.c
@@ -940,7 +940,7 @@ char            fullpath[_MAX_PATH];
     } else if (var_DISPLAY_PROG == L_DISPLAY_WITH_OPEN) {
         snprintf(buffer, L_BUF_SIZE, "open %s &", tempname);
     }
-    ignore = system(buffer);
+    //ignore = system(buffer);
 
 #else  /* _WIN32 */
 
@@ -1010,7 +1010,7 @@ char     fullpath[_MAX_PATH];
     FREE(tail);
 #endif  /* _WIN32 */
 
-    ignore = system(buffer);  /* gthumb || i_view32.exe */
+    //ignore = system(buffer);  /* gthumb || i_view32.exe */
     return 0;
 }
 
diff --git a/src/xtractprotos.c b/src/xtractprotos.c
index e5ebe89..faaae4f 100644
--- a/src/xtractprotos.c
+++ b/src/xtractprotos.c
@@ -83,6 +83,7 @@
  */
 
 #include <string.h>
+#include <sys/errno.h>
 #include "allheaders.h"
 
 static const l_int32  L_BUF_SIZE = 512;
@@ -190,7 +191,8 @@ static char  mainName[] = "xtractprotos";
 	    continue;
 	snprintf(buf, L_BUF_SIZE, "cpp -ansi -DNO_PROTOS %s %s",
 	         filein, tempfile);
-	ret = system(buf);  /* cpp */
+	//ret = system(buf);  /* cpp */
+	ret = ENOENT;		// For iOS compat, claim ENOENT
 	if (ret) {
             fprintf(stderr, "cpp failure for %s; continuing\n", filein);
 	    continue;
